docs/homestead_api.md

```diff
@@ -107,5 +107,5 @@ Response:
 
 `{ "result-code": "2001", "mandatory-capabilties": [], "optional-capabilities": [] }`
 
-* 404 if the user cannot be found.
+* 404 if the user cannot be found (either a 5001 error from the HSS, or having no HSS configured and no record of the user).
```

Comment:
It would be nice to split out the 200 response code a bit more (e.g. the response won't contain a real HSS result code if there's no HSS)